src/attribution/onINP.ts

```diff
@@ -90,10 +279,26 @@ export const onINP = (
   onReport: INPReportCallbackWithAttribution,
   opts?: ReportOpts,
 ) => {
+  if (!loafObserver) {
+    loafObserver = observe('long-animation-frame', handleLoAFEntries);
+  }
+  if (!eventObserver) {
+    eventObserver = observe('event', groupEntriesByRenderTime, {
+      durationThreshold: opts!.durationThreshold ?? 40,
```

Comment:
We've hardcoded that `40` in two places now. Could we import it from the regular `onINP` rather than duplicating it here?

Reply:
Fair point. I don't want to extend the public API of `onINP.js` but I'll find a shared place to import it from.